node/sc/sub_event_handler.go

```diff
@@ -45,6 +45,24 @@ func (cce *ChildChainEventHandler) HandleChainHeadEvent(block *types.Block) erro
 	logger.Trace("bridgeNode block number", "number", block.Number())
 	cce.handler.LocalChainHeadEvent(block)
 
+	go func() {
```

Comment:
Please leave a comment here about the reason for using goroutine to fetch `parentOperatorBalance`.

Reply:
Thanks, I added.